integration-tests/docker/test_env/cl_node.go

```diff
@@ -411,19 +412,21 @@ func (n *ClNode) getContainerRequest(secrets string) (
 	}, nil
 }
 
-func GetOracleIdentities(chainlinkNodes []*ClNode) ([]int, []confighelper.OracleIdentityExtra) {
+func GetOracleIdentities(chainlinkNodes []*ClNode) ([]int, []confighelper.OracleIdentityExtra, error) {
```

Comment:
The lint changes required a decent changeset for this one. @lukaszcl Does this keep the intended functionality?

Reply:
@tateexon it looks good I think. Maybe add a test to check it? I have a similar function in Mercury tests which I use. Not if this one is used anywhere.

Reply:
Did a bunch of digging. Looks like you have an almost identical one in both mercurys that was recently edited. It is also copy pasted within chainlink itself but with some added functionality that is needed for automation tests 🤦 Ripping it out of here since we don't need yet another copy paste sitting around.